<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskCreateDialog</class>
 <widget class="QDialog" name="TaskCreateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>453</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>编辑任务</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QDialog {
      background-color: #222;
      color: white;
    }
    QLabel, QRadioButton, QCheckBox {
      color: white;
    }
    QGroupBox {
      border: 1px solid #666;
      margin-top: 10px;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="titleLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>标题:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="descLayout">
     <item>
      <widget class="QLabel" name="descLabel">
       <property name="text">
        <string>描述:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="descEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="priorityGroup">
     <property name="title">
      <string>优先级</string>
     </property>
     <layout class="QHBoxLayout" name="priorityLayout">
      <item>
       <widget class="QRadioButton" name="lowRadio">
        <property name="text">
         <string>低</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="mediumRadio">
        <property name="text">
         <string>中</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="highRadio">
        <property name="text">
         <string>高</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="dateLayout">
     <item>
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>截止日期:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="dateEdit">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="date">
        <date>
         <year>2025</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dateSelectBtn">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="completedCheck">
     <property name="text">
      <string>已完成</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>保存</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
